_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -38,10 +41,10 @@ This Quick Start creates a new GKE cluster, installs Istio and then deploys the
 
 1. Once you have an account and project enabled, click the following link to open the Deployment Manager.
 
-   - [Istio GKE Deployment
-Manager](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin){:target="_blank"}
+   - [Istio GKE Deployment Manager](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin)
 
-   We recommend that you leave the default settings as the rest of this tutorial shows how to access the installed features. By default the tool creates a GKE alpha cluster with the specified settings, then installs the Istio [control plane](https://istio.io/docs/concepts/what-is-istio/overview.html#architecture){:target="_blank"}, the [BookInfo](https://istio.io/docs/guides/bookinfo.html){:target="_blank"} sample app, [Grafana](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html){:target="_blank"} with [Prometheus](https://istio.io/docs/tasks/telemetry/querying-metrics.html){:target="_blank"}, [ServiceGraph](https://istio.io/docs/tasks/telemetry/servicegraph.html){:target="_blank"}, and [Zipkin](https://istio.io/docs/tasks/telemetry/distributed-tracing.html#zipkin){:target="_blank"}. You'll find out more about how to access all of these below.
+   We recommend that you leave the default settings as the rest of this tutorial shows how to access the installed features. By default the tool creates a 
+   GKE alpha cluster with the specified settings, then installs the Istio [control plane]({{home}}/docs/concepts/what-is-istio/overview.html#architecture), the [BookInfo](https://istio.io/docs/guides/bookinfo.html) sample app, [Grafana](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html) with [Prometheus](https://istio.io/docs/tasks/telemetry/querying-metrics.html), [ServiceGraph](https://istio.io/docs/tasks/telemetry/servicegraph.html), and [Zipkin](https://istio.io/docs/tasks/telemetry/distributed-tracing.html#zipkin). You'll find out more about how to access all of these below.
```

Comment:
looks like there are more here. need to add “g” at the end of you sed expression?